scheduler/models/queue.py

```diff
@@ -53,13 +53,14 @@ class PrioritizedItemORM(Base):
     created_at = Column(
         DateTime(timezone=True),
         nullable=False,
-        default=datetime.datetime.utcnow,
+        default=datetime.utcnow,
     )
+
     modified_at = Column(
         DateTime(timezone=True),
         nullable=False,
-        default=datetime.datetime.utcnow,
-        onupdate=datetime.datetime.utcnow,
+        default=datetime.utcnow,
+        onupdate=datetime.utcnow,
```

Comment:
From what I understand the best way to do this would be to use `func.now()`, see https://stackoverflow.com/a/33532154.

Reply:
awesome thanks